src/main/java/com/tritoncubed/launchapi/Payload.java

```diff
@@ -31,50 +31,196 @@
     public float getTemp_C() { return temp_C; }
     public void setTemp_C(float temp_C) { this.temp_C = temp_C; }
     
+    
+    /**
+     * Creates static instance of enum 
+     */
+    public static final FS_XL fs_xl = FS_XL.FS_XL_2;
+    
+    /**
+     *  Determines the scale and precision of the accelerometer
+     *  g is the standard acceleration due to Earth's gravity
+     *  Conversion to float from conversion factor of enum. 
+     */
+    public enum FS_XL {
+    	FS_XL_2(0.061f),
+    	FS_XL_4(0.122f),
+    	FS_XL_8(0.244f),
+    	FS_XL_16(0.732f);
+    	
+    	public final float conversionFactor;
+    	
+    	private FS_XL(float convFact) {	
+    		this.conversionFactor = convFact * 9.80665f / 1000;
+    	}	
+    }
+    
+    /**
+     * Determines the scale and precision of the gyroscope 
+     * Conversion to float from conversion factor of enum. 
+     */
+    public enum FS_G {
+    	FS_G_245(8.75f),
+    	FS_G_500(17.5f),
+    	FS_G_2000(70.0f);
+    	
+    	public final float conversionFactor;
+    	
+    	private FS_G(float convFact) {
+    		this.conversionFactor = convFact / 1000;
+    	}
+    }
+    
+    /**
+     * Determines the scale and precision of the magnetosensor
+     * Conversion to float from conversion factor of enum. 
+     */
+    enum FS_M {
+    	FS_M_4(0.14f),
+    	FS_M_8(0.29f),
+    	FS_M_12(0.43f),
+    	FS_M_16(0.58f);
+    	
+    	public final float conversionFactor;
+    	
+    	private FS_M(float convFact) {
+    		this.conversionFactor = convFact / 1000;
+    	}
+    }
+    
     /**
      * Payload Constructor 
      * @param accel_x acceleratoin on x axis
      * @param accel_y acceleratoin on y axis 
      * @param accel_z acceleration on z axis
+     * @param gyro_x angular motion in the x direction
+     * @param gyro_y angular motion in the y direction
+     * @param gyro_z angular motion in the z direction
+     * @param mag_x magnetic field in the x direction
+     * @param mag_y magnetic field in the y direction
+     * @param mag_z magnetic field in the z direction
      * @param temp_c temperature from payload
      */
-    public Payload(short accel_x, short accel_y, short accel_z, float temp_c) {
-    	this.accel_X = accel_x;
-    	this.accel_Y = accel_y;
-    	this.accel_Z = accel_z;
+    public Payload(short accel_x, short accel_y, short accel_z, 
+    short gyro_x, short gyro_y, short gyro_z, short mag_x, 
+    short mag_y, short mag_z, float temp_c) {
+    	
+    	this.accel_X = accel_x * fs_xl.conversionFactor;
+    	this.accel_Y = accel_y * fs_xl.conversionFactor;
+    	this.accel_Z = accel_z * fs_xl.conversionFactor;
+    	this.gyro_X = gyro_x * fs_xl.conversionFactor;
+    	this.gyro_Y = gyro_y * fs_xl.conversionFactor;
+    	this.gyro_Z = gyro_z * fs_xl.conversionFactor;
+    	this.mag_X = mag_x * fs_xl.conversionFactor;
+    	this.mag_Y = mag_y * fs_xl.conversionFactor;
+    	this.mag_Z = mag_z * fs_xl.conversionFactor;
     	this.temp_C = temp_c;
     }
     
+    /**
```

Comment:
Don't start comments with "Method that..." or Function that...", it is implied.

Also, avoid use of "print" unless specifically using any of the Print/Writers/Logger/Builder objects. Instead use "generates"